Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Color-Picker/index.md

```diff
@@ -46,4 +46,45 @@ The Color picker allows you to set some predetermined colors that the editor can
         <div style="background-color: #@hexColor">@colorLabel</div>
     }
 }
-```
\ No newline at end of file
+```
+
+## Add values programmatically
+
+See the example below to see how a value can be added or changed programmatically. To update a value of a property editor you need the [Content Service](../../../../../Reference/Management/Services/ContentService/index.md).
+
+```csharp
+@{
+	// Get access to ContentService
+	var contentService = Services.ContentService;
+
+	// Create a variable for the GUID of the page you want to update
+	var guid = Guid.Parse("32e60db4-1283-4caa-9645-f2153f9888ef");
+
+	// Get the page using the GUID you've defined
+	var content = contentService.GetById(guid); // ID of your page
+
+	// Set the value of the property with alias 'color'. 
+	content.SetValue("color", "38761d");
```

Comment:
Should we mention that the value set here, needs to be one of the prevalues on the Color Picker in order to work? 🤔
It might be different when you have it in a complete setup, but testing this sample, it won't set anything unless it's one of the prevalues I've set.

What do think?

Reply:
@sofietoft Good point. I added a comment